doc/UsersGuide/source/FAQ.rst

```diff
@@ -150,10 +150,314 @@ places, and depends on whether you are using the write component to generate you
 ==============================================================
 How do I set the total number of tasks for my job?
 ==============================================================
-The total number of MPI tasks used by the UFS Weather Model is a combination of compute and quilt tasks, and can be calculated using the following relationship:
 
-   * total tasks = ``compute tasks + quilt tasks``
-   * compute tasks = ``x layout * y layout * number of tiles``
-   * quilt tasks = ``write_groups * write_tasks_per_group`` if ``quilting==.true.``
+In the UFS WM, each component's MPI task information, including the
+starting task and end ending task numbers and the number of threads,
```

Comment:
'_starting task and end ending task numbers and the number of threads,_'

extra 'end'

and I think instead of 'task numbers' it's better to say 'task ranks'